### PR TITLE
Uninstall pulp_docker rpms

### PR DESCRIPTION
**Pull request by asmacdo**
The job builder should install these packages from source rather than
from an rpm.

https://pulp.plan.io/issues/3744
fixes: #3744

### CODE REVIEW

File: ci/jjb/README.md [hunk 1]
```diff
@@ -129,18 +129,26 @@ pack JJB will complain about SSL issues. To fix this add the CA to your system
...
-To workaround SSL errors, patch line 431 of
-`/usr/lib/python2.7/site-packages/jenkins/__init__.py`:
+
+To workaround SSL errors, you'll need to patch jenkins.
 
 ```python
-import ssl
-context = ssl._create_unverified_context()
-response = urlopen(req, context=context, timeout=self.timeout).read()
+(jjb) [vagrant@pulp3 pulp-ci]$ sed -n 522,530p  ~/.virtualenvs/jjb/lib/python2.7/site-packages/jenkins/__init__.py
```
Comment (Ichimonji10) [inline ci/jjb/README.md]: This ain't Python code!

**Comment by Ichimonji10** [comment]
This PR now does much more than uninstalling pulp_docker RPMs. Is there an over-arching theme to these changes? Or are the four commits in this PR separate, and bundled together for the author's convenience?

**Comment by asmacdo** [comment]
@Ichimonji10, I've just been fixing the tests and doc builders, discovering new issues as I went. The over-arching theme is I wanted my checks to pass for https://github.com/pulp/pulp_docker/pull/235, but they really are separate changes.

File: ci/jjb/README.md [hunk 2]
```diff
@@ -129,18 +129,26 @@ pack JJB will complain about SSL issues. To fix this add the CA to your system
...
+        _settings['timeout'] = self.timeout
+        return self._session.send(r, **_settings)
 ```
+
 This workaround was tested using the following packages versions:
 
- - python-jenkins==0.4.16
- - jenkins-job-builder==2.0.2
+ - jenkins-job-builder==2.0.9
+ - python-jenkins==1.0.1
```
[the remark this answers is not in the document]
Comment (asmacdo) [reply ci/jjb/README.md]: Yup, I ran the update to test the changes.

**Pull request merged**